x/leverage/keeper/collateral.go

```diff
@@ -19,7 +19,7 @@ func (k Keeper) liquidateCollateral(ctx sdk.Context, borrower, liquidator sdk.Ac
 // burnCollateral removes some uTokens from an account's collateral and burns them. This occurs
 // during liquidations.
 func (k Keeper) burnCollateral(ctx sdk.Context, addr sdk.AccAddress, uToken sdk.Coin) error {
-	err := k.setCollateralAmount(ctx, addr, k.GetCollateralAmount(ctx, addr, uToken.Denom).Sub(uToken))
+	err := k.setCollateral(ctx, addr, k.GetCollateral(ctx, addr, uToken.Denom).Sub(uToken))
```

Comment:
I was wondering if it is worth creating a `k.subCollateral`? 
But it was only used a couple of times to subtract from the current collateral